<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>goods_manage_window</class>
 <widget class="QDialog" name="goods_manage_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoodsManage</string>
  </property>
  <widget class="QTableWidget" name="table_g">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>451</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>179</number>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="search_b">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="code_l">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>470</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Count :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_b">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Code :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="count_l">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="done_b">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>470</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="new_b">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NEW</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
